README.md

```diff
@@ -306,13 +306,36 @@ callbacks have been executed. After this, the disconnection callback is executed
 There are a few hooks that need to be set on the cluster context object after it is created.
 See the `adapters/` directory for bindings to *ae* and *libevent*.
 
-## AUTHORS
+### Allocator injection
+
+Hiredis-cluster uses hiredis allocation structure with configurable allocation and deallocation functions. By default they just point to libc (`malloc`, `calloc`, `realloc`, etc).
+
+#### Overriding
 
-This fork is based on the heronr fork (https://github.com/heronr/hiredis-vip)
-and uses hiredis (https://github.com/redis/hiredis).
+If you have your own allocator or if you expect an abort in out-of-memory cases,
+you can configure the used functions in the following way:
+
+```c
+hiredisAllocFuncs myfuncs = {
+    .mallocFn = my_malloc,
+    .callocFn = my_calloc,
+    .reallocFn = my_realloc,
+    .strdupFn = my_strdup,
+    .freeFn = my_free,
+};
+
+// Override allocators (function returns current allocators if needed)
+hiredisAllocFuncs orig = hiredisSetAllocators(&myfuncs);
+```
 
-Hiredis-vip was originally created by vipshop (https://github.com/vipshop/hiredis-vip).
+To reset the allocators to their default libc functions simply call:
+
+```c
+hiredisResetAllocators();
+```
+
+## AUTHORS
 
-The Redis Cluster client library part in hiredis-vip was written by deep (https://github.com/deep011).
+This project is used and sponsored by Ericsson. It is a fork of the now unmaintained [hiredis-vip](https://github.com/vipshop/hiredis-vip).
```

Comment:
Maybe mention the heronr improvements and other things we have taken in? Or we simply link to CHANGELOG.md

"used and sponsored" is interesting, but not very relevant under Authors. It's already in the About text for the fork.

Perhaps we can mention something which fits better under Authors, such as "maintained and improved by Ericsson Software Technology (EST)".

If we don't mention much about authors, we can rename "AUTHORS" to something else?

Reply:
maybe we should remove it, I think its copied from hiredis where they got a AUTHORS section